job_creator.php

```diff
@@ -242,7 +242,7 @@ private function isAllowedPhpVersion(string $phpVersion)
     private function getInstallerBranch(): string
     {
         $version = str_replace('.x-dev', '', $this->installerVersion);
-        if (!in_array($this->repoName, FORCE_INSTALLER_REPOS) && isset($this->repoData['type']) && $this->repoData['type'] === 'recipe') {
+        if (in_array($this->repoName, NO_INSTALLER_REPOS) || !in_array($this->repoName, FORCE_INSTALLER_REPOS) && isset($this->repoData['type']) && $this->repoData['type'] === 'recipe') {
```

Comment:
Could you put an extra set of brackets around something here so that it's clear if the || or the && has precedance

Reply:
Done